userdetails-cognito/src/main/groovy/au/org/ala/userdetails/CognitoUserService.groovy

```diff
@@ -504,42 +520,100 @@ class CognitoUserService implements IUserService {
 
     @Override
     void findScrollableUsersByUserName(String username, int maxResults, ResultStreamer resultStreamer) {
-        throw new NotImplementedException()
+        streamUserResults(resultStreamer, listUsers(username, null, maxResults))
```

Comment:
Shouldn't this be paging through the listUsers cognito call and offering each page to the stream?

Reply:
Shall I address this is in the clean up branch since there are few new changes here in clean up branch?

Reply:
Sounds good